Dockerfile

```diff
@@ -26,8 +26,20 @@ COPY --chown=1000:1000 . /bioformats2raw
 WORKDIR /bioformats2raw
 RUN gradle build
 RUN cd build/distributions && rm bioformats2raw*tar && unzip bioformats2raw*zip && rm -rf bioformats2raw*zip
+
+
+FROM openjdk:8 as final
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update -y -q \
+ && apt-get install -y --no-install-recommends -q libzeroc-ice3.7-java libblosc1 \
+ && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+
 USER root
-RUN mv /bioformats2raw/build/distributions/bioformats2raw* /opt/bioformats2raw
+COPY --from=build /bioformats2raw/build/distributions/bioformats2raw* /opt/bioformats2raw
+ENV PATH="/opt/bioformats2raw/bin:${PATH}"
+
 USER 1000
 WORKDIR /opt/bioformats2raw
-ENTRYPOINT ["/opt/bioformats2raw/bin/bioformats2raw"]
+CMD ["bioformats2raw"]
```

Comment:
@ilveroluca, what's the benefit of CMD over ENTRYPOINT here? With CMD,

```
docker run -ti --rm josh-bf2raw-pr123 -h
```
fails.

Reply:
Indeed, because this image does not specify an entrypoint on needs to specify the entire command line when launching the container, i.e.:
```
docker run -ti --rm josh-bf2raw-pr123 bioformats2raw -h
```

ENTRYPOINT vs CMD is often a matter of debate.  The main benefit of not specifying the entrypoint is in terms of flexibility.  The image becomes more easily usable as an environment in which to run bioformats2raw and also other related commands (naturally, this could also be done by overriding the entrypoint...).  However, it does come at the cost of usability.

The concrete use case that led me to make this change is actually gone now.  I needed to run something like
```
mkdir -p /multi/level/output/path/ && bioformats2raw ... /multi/level/output/path/slide.raw
```
in an image conversion workflow I wrote with Snakemake, and being able to run the whole command within the same container made things a bit simpler for the workflow.  However, I have since learned that Snakemake uses container images (including Docker images) by opening a `singularity shell` and thus completely ignores both ENTRYPOINT and CMD.

At this point, I would be inclined to restore the original ENTRYPOINT for simplicity.

Reply:
:+1: